### DISCUSSION TITLE
poppler-qt6

### DISCUSSION THREAD

**Discussion by sunderme** [Homebrew/discussions#1771] [Everyday usage] [answered]
Is there a plan to provide poppler-qt6 ? 
Currently only poppler-qt5 is offered, though poppler itself offers a apparently stable qt6 interface.
This is the block to move towards qt6 for texstudio and similar apps.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew

**Comment by carlocab** [comment, marked as answer]
I believe the difficulty is that while `poppler` itself supports `qt6`, many formulae that depend on `poppler` only support `qt5`.

I wouldn't be opposed to switching `poppler` to `qt6` and adding a `poppler-qt5` formula for all the dependents that still need `qt5`.

**Reply by sunderme** [reply]
sounds fine for me.
In the past, formula options were used for this (-with-qt5 if I remember correctly). Not sure if that is still supported or desired.

**Reply by carlocab** [reply]
We no longer allow options in Homebrew/core (they lead to configurations that aren't tested in CI and require users to build from source), so a separate formula is probably the way to get `qt6` support for `poppler`.

**Reply by sunderme** [reply]
So, how do we proceed ?

**Reply by carlocab** [reply]
You can open a pull request that implements what I suggested above. Or, you can wait to see if someone else will pick it up, but that might take a while.

**Reply by gromgit** [reply]
> Or, you can wait to see if someone else will pick it up, but that might take a while.

On it, boss! :laughing: 

EDIT: Oh, never mind, @sunderme's on it.
